src/util/templates.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"bytes"
+	"text/template"
+)
+
+// Helper function to parse, execute and convert "text/template" to string. Panics on error.
+func TemplateToString(format string, data interface{}) string {
```

Comment:
I would remove this file and put this function in terminal_test.go at the moment.